### ISSUE TITLE
No está en play store?

### ISSUE THREAD

**Issue by ghost** [Onelio/ConnectU#23]

**Comment by dgrr** [comment]
No. La denunciaron y la han eliminado. No sabemos quien podra ser....

**Comment by JoeMK4** [comment]
Esta app es mucho mejor tanto en interfaz como en rapidez que la nueva, ganas de joder, espero que siga adelante...

**Comment by dgrr** [comment]
Por el momento la app no va a seguir adelante. Por lo visto @Onelio ha tenido muchos problemas con la app porque la UA le ponia restricciones, y al parecer han denunciado la app (no quiero acusar directamente a nadie).
Aun asi, aceptamos contribuciones a este repositorio. Tampoco espereis que podamos hacer mucho. @Onelio se ha cansado de estar evadiendo sus restricciones y yo ni estoy en la UA, por lo que no es practico para mi.
